_common/GFMAlerts.php

```diff
@@ -16,38 +16,34 @@
 
 class GFMAlerts extends Parsedown
 {
-    public $VERSION = '1.0';
-    
-    public function __construct()
-    {
-       $this->BlockTypes['>'][] = 'Alert';
-    }
+    private $icon = array(
```

Comment:
I moved the icons into a private property so that they weren't being created again each call to `blockQuote()`